http_service/bugbug_http/models.py

```diff
@@ -61,7 +61,7 @@ def get_model(model_name):
         # Cache the model only if it was last used less than two hours ago.
         if model_name in MODEL_LAST_LOADED and MODEL_LAST_LOADED[
```

Comment:
Is `MODEL_LAST_LOADED` required anymore now that TTLCache has a `items_last_touched`? 
You could just write an API in TTLCache 
```python
def last_touched_time(self, item):
    return self.items_last_touched[item]
```

and replace the above line with:
```python
if model_name in MODEL_CACHE and MODEL_CACHE.last_touched_time(model_name)
```